recipe/meta.yaml

```diff
@@ -30,51 +31,49 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - patch   # [not win]
+    - m2-patch    # [win]
   host:
+    - pip
+    - python
     - cython
+    - docutils
     - numpy
-    - pip
+    - recommonmark
     - setuptools
+    - sphinx
     - wheel
-    - python
-    - recommonmark
-    - sphinx >=4.2.0
-    - docutils <0.17  # docutils changed html template in 0.17
   run:
-    # appnope should be in orange-widget-base
-    - appnope  # [osx]
-    - anyqt >=0.1.0
+    - anyqt >=0.2.0
     - baycomp >=1.0.2
     - bottleneck >=1.3.4
+    - catboost >=1.0.1
     - chardet >=3.0.2
-    - httpx >=0.21
-    - joblib >=0.11
+    - httpx >=0.21.0
+    - joblib >=1.0.0
     - keyring
     - keyrings.alt
     - matplotlib-base >=3.2.0
     - networkx
-    - numpy >=1.19.5
+    - numpy >=1.20.0
     - openpyxl
     - opentsne >=0.6.1,!=0.7.0
-    - orange-canvas-core >=0.1.28,<0.2a
-    - orange-widget-base >=4.19.0
-    - pandas >=1.3.0,!=1.5.0
+    - orange-canvas-core >=0.1.30,<0.2a
+    - orange-widget-base >=4.22.0
+    - pandas >=1.4.0,!=1.5.0,!=2.0.0
     - pip >=18.0
```

Comment:
I wonder why `pip` in `run`. Seems there was an upstream issue to `install Orange on Ubuntu 14.04` https://github.com/biolab/orange3/commit/959a79e14344856da534d7dc1bc0b4f1f92aaaa4. Does it impact us also?

Reply:
Unfortunately, I had some issues getting my environment setup, so I could not test that.

Reply:
But I did leave it because it was listed [here](https://github.com/biolab/orange3/blob/3.36.2/requirements-core.txt#L16)